fulp_modules/devils/devil.dm

```diff
@@ -0,0 +1,382 @@
+#define DEVIL_BODYPART "devil"
+#define DEVIL_HANDS_LAYER 1
+#define DEVIL_TOTAL_LAYERS 2
+
+/mob/living/carbon/true_devil
```

Comment:
oh lord, you made true devils subtypes of carbons but not of humans again?

this could cause issues in the future...